SalesforceMobileSDK-iOS.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
       smartstore.dependency 'SalesforceMobileSDK-iOS/SalesforceSDKCore'
       smartstore.dependency 'SQLCipher', '~> 3.1'
       smartstore.dependency 'SQLCipher/fts', '~> 3.1'
-      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h', 'external/fmdb/src/fmdb/*.{h,m}'
+      smartstore.dependency 'FMDB', '~> 2.5'
+      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h'
```

Comment:
The other change that seems like it was lost is the inclusion of `-DSQLITE_HAS_CODEC`.  That needs to be included somewhere.